scripts/tasks/api-extractor.ts

```diff
@@ -1,6 +1,8 @@
+import fs from 'fs';
 import * as glob from 'glob';
 import * as path from 'path';
-import { apiExtractorVerifyTask, task, series } from 'just-scripts';
+import jju from 'jju';
```

Comment:
Q: Any specific reason why we use `jju`?

Reply:
good question!

this is approach we do so json files with comments don't break - although not every tooling supports json5 we still do it. nx for example uses different library under the hood that we might use as well once we migrate more towards it in the future....

Reply:
Ah, wasn't aware we were using json5 features. Thanks for clarifying!